RW: running simulations on the cluster

debapratimj opened the pull request:
-  Alpha convention fixed
-  HTML fig saving 
- Added first passage time, step size stats
- Hyperparameters for 3 coupling/social info quality cases 
- Changes  in agent.py, model.py to add some functionalities

debapratimj committed:
Added first catch, step size distribution , fixed alpha convention and optuna db

debapratimj committed:
Updated social information filtering scripts

debapratimj committed:
Correct hyperparamters for experiment

debapratimj committed:
change number of parallel jobs for running on cluster

debapratimj committed:
further reduce num_jobs

debapratimj committed:
changed to optuna results in html saving

vagechirkov commented:
Hey @debapratimj, thanks for the updates! Please exclude results (database) from PR

debapratimj committed:
Removing plotting for debug

vagechirkov committed:
simplification of time to first catch recording

vagechirkov committed:
Refactor distance metrics and add Manhattan distance tracking

Replaced `_traveled_distance` with separate `euclidean` and `manhattan` distance metrics. Updated agent movement logic to compute distances using `scipy.pdist`, ensuring consistency and accuracy. Added new tests to validate both distance calculations and step size updates.

vagechirkov committed:
Add *.html and *.db to .gitignore

vagechirkov committed:
Remove redundant path insertion and add Python path setup notes; formatting clean-up

vagechirkov committed:
Streamline README and update requirements

vagechirkov commented:
Request for changes:
- Separate experiment execution and plotting;
- modularize the plotting code (i.e., make the plotting function for the non-optuna figures)
- add study name to `optuna.create_study` for easier navigation in the dashboard

vagechirkov commented on experiments/experiment_071224_rw_default_no_coupling.py:
```diff
@@ -73,57 +78,81 @@ def objective(trial):
 
-    # Visualization: Plot the optimization history
-    import plotly.io as pio
+    if plotting:
```
For this and all other experiments, all code below this line should not be in this file

debapratimj replied:
acceoted changes

vagechirkov reviewed (changes requested):
- Separate experiment execution and plotting;
- modularize the plotting code (i.e., make the plotting function for the non-optuna figures)
- add study name to `optuna.create_study` for easier navigation in the dashboard

vagechirkov committed:
remove tick labels

debapratimj committed:
Stashing changes in default_filtering script

debapratimj committed:
stashing changes in default filtering

debapratimj committed:
Added separate code for visualizations, modified optuna saving

debapratimj committed:
fixed some minor issues in experiment runners, tested before running on cluster

debapratimj committed:
Scripting changes to visualization, objective function

debapratimj committed:
optuna saving updates, visualization for movements updated

debapratimj committed:
Added parameter sweep and viz

debapratimj committed:
batch run parallelisation

Commit:
Cloud parallezation, increased resource amount, catch distributions and solved very few samples in histogram plots issue

Commit:
30% consumption rule and removed param results

Commit:
30% consumption rule and removed param results

vagechirkov committed:
Remove CSV and JSON files from tracking and update .gitignore

vagechirkov committed:
fix the test

Commit:
cleaned up repo, removed old parameter results
